built-in-references/Kubernetes/allowed-external-ips/template.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazureexternalips
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureExternalIPs
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            allowedIPs:
+              type: array
+              items:
+                type: string
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8sexternalips
```

Comment:
Rename the package name as 'k8sazureexternalips' matching with the template name.

Reply:
done